### DISCUSSION TITLE
Where can I find hard-upload-errors-log.txt

### DISCUSSION THREAD

**Discussion by jpbraun** [GetPublii/Publii#1832] [Bug Report]
### Operating system

Windows 10 / Windows 11

### Publii version

0.46.2

### Issue type

Application

### Bug description

Synchronizing fails. Message: "Einige Daten wurden nicht korrekt synchronisiert. Bitte überprüfen Sie die Datei hard-upload-errors-log.txt mit Tool -&gt; Log-Viewer-Tool." Unfortunately there is no file with the name "hard-upload-errors-log.txt".

**Comment by michaelfluegge** [comment]
Hi, @jpbraun , I googled that "hard-upload-errors-log.txt" for you and found https://github.com/GetPublii/Publii/issues/597. Maybe @dziudek can say something about this issue

**Comment by dziudek** [comment]
@jpbraun - please check if under deployment logs do you have some lines containing `UPL HARD ERR` - I will check in v.0.46.3 or v.0.47 the logging - I suppose it requires some improvements
